ontopy/ontology.py

```diff
@@ -76,18 +76,32 @@ def __init__(self, *args, **kwargs):
         self._iri_mappings = {}  # all iri mappings loaded so far
         super().__init__(*args, **kwargs)
 
-    def get_ontology(self, base_iri="emmo-inferred"):
+    def get_ontology(
+        self,
+        base_iri: str = "emmo-inferred",
+        OntologyClass: "owlready2.Ontology" = None,
+        label_annotations: "Sequence" = None,
+    ) -> "Ontology":
+        # pylint: disable=too-many-branches
         """Returns a new Ontology from `base_iri`.
 
-        The `base_iri` argument may be one of:
-          - valid URL (possible excluding final .owl or .ttl)
-          - file name (possible excluding final .owl or .ttl)
-          - "emmo": load latest version of asserted EMMO
-          - "emmo-inferred": load latest version of inferred EMMO
-            (default)
-          - "emmo-development": load latest inferred development version
-            of EMMO. Until first stable release emmo-inferred and
-            emmo-development will be the same.
+        Arguments:
+            base_iri: The base IRI of the ontology. May be one of:
+                - valid URL (possible excluding final .owl or .ttl)
+                - file name (possible excluding final .owl or .ttl)
+                - "emmo": load latest version of asserted EMMO
+                - "emmo-inferred": load latest version of inferred EMMO
+                  (default)
+                - "emmo-development": load latest inferred development
+                  version of EMMO. Until first stable release
+                  emmo-inferred and emmo-development will be the same.
+            OntologyClass: If given and `base_iri` doesn't correspond
+                to an existing ontology, a new ontology is created of
+                this Ontology subclass.
```

Comment:
What is the default?

Reply:
The default for OntologyClass is ontopy.Ontology

Reply:
Kind of, but as it is written now it is None, and if it is None it is then made to ontopy.Ontology. I don't really care how it is done, but at least it should be clear in the documentation what the default is (and now it seems like none).

Reply:
Ok, documented the default.